src/main/java/io/github/fablabsmc/fablabs/api/gamerule/v1/FabricGameRuleCategory.java

```diff
@@ -0,0 +1,32 @@
+package io.github.fablabsmc.fablabs.api.gamerule.v1;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+
+/**
+ * Utility class for registering GameRule objects with custom categories outside of the seven categories MC provides
+ */
+public class FabricGameRuleCategory implements Comparable<FabricGameRuleCategory>{
+    private final Text category;
+
+    public FabricGameRuleCategory(String category) {
+        this.category = new LiteralText(category).styled((style) -> style.withBold(true).withColor(Formatting.YELLOW));
+    }
+
+    public FabricGameRuleCategory(Text category) {
+        this.category = category;
+    }
+
+    @Environment(EnvType.CLIENT)
```

Comment:
I'd say drop the `EnvType.CLIENT` here even though the actual `RuleCategory` has a client only `getCategory() -> String` method.